tests/Repository/SubmitRepositoryTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Tests\Repository;
+
+use App\Entity\Submit;
+use App\Repository\SubmitRepository;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+class SubmitRepositoryTest extends KernelTestCase
+{
+    public function testUpdateDoneSubmitsWork()
```

Comment:
This test does not really work: You have no guarantee that some `pastAcceptedSubmitsCount` is > 0 before the test start. So this test does not prove values has been updated.

But I guess it works with fresh fixtures, so it kindy works in the CI but not in dev. 

Anyway, I'm gonna merge this PR. But this is something you have to think about

1. what are my initial data (you must be sure what they are, and you can't rely on fixtures only, since another tests may have modified it -- or you need a strong policy about fixuere 
2. do something
3. assert it's OK

Some ref:
* https://jolicode.com/blog/ameliorer-la-dx-de-vos-fixtures-php
* http://agileinaflash.blogspot.com/2009/02/first.html

Reply:
This test wasn't really made to test that values have been updated, but rather to ensure that after running the command, no submits are left with an accepted status if their conference has ended.

But you're right, since we may have 0 submits in that case due to fixtures, the tests could pass while the command is actually broken, so I'll improve this test and make sure it actually updates some data. And since I wrote several tests that rely on fixtures data, I will make an issue to improve this.

Reply:
> This test wasn't really made to test that values have been updated, but rather to ensure that after running the command, no submits are left with an accepted status if their conference has ended.

I should have :)